designer/server/src/routes/library.test.js

```diff
@@ -13,6 +17,11 @@ describe('Form library routes', () => {
   const htmlContentType = 'text/html'
 
   test('Testing form library list page', async () => {
+    const title = 'Form 1'
+
+    // Mock the api call to forms-manager
+    forms.list.mockResolvedValueOnce([{ title }])
```

Comment:
Love seeing all these tests go green! 🙌

@davidjamesstone Do you mind changing `forms.list` → `jest.mocked(forms.list)`?

```suggestion
    jest.mocked(forms.list).mockResolvedValueOnce([{ title }])
```

<br>

It enables autocomplete for Jest methods but also type checks the arguments

<img width="818" alt="Using jest mocked" src="https://github.com/DEFRA/forms-designer/assets/415517/8fed7de8-44d4-4b08-91e5-cfb6cc5eae03">

Reply:
Done - thanks.